source/scripts/user.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * This file contains functions to create a user account, store the user account to local storage,
+ * login to a user account that help manipulate the account system. A User class might not be needed
+ * to create a general object for a user to be stored in local storage.
+ * -------------------------------------
+ * Some stretch goals:
+ * - delete an account and delete all the posts on that account (?)
+ * - retype password to confirm when creating an account
+ * - "Forgot password" option in login page
+ * - when someone enters the page containg the domain of our app, redirect to login page if not logged in
+ */
+
+// constant urls
+const APP_MAIN_URL = "some_url_for_the_main_app_page";
+const CREATE_ACCOUNT_URL = "some_url_for_user_to_create_account";
+const LOGIN_URL = "some_url_for_user_to_login_in";
+
+/**
+ * Class for creating User type Object, could instead just declare a general Object type without this
+ */
+class User extends HTMLElement() {

```

Comment:
This class probably doesn't need to extend HTMLElement, since it isn't an element on the HTML page

Reply:
Sorry, I was held up by other assignments and midterms last week and didn't notice your comments. These suggestions all make sense and I have updated user.js accordingly. I will open another pull request with more detailed info.